backend/src/controllers/orders-controller.ts

```diff
@@ -80,10 +89,26 @@ export const completeOrder = async (req: Request, res: Response) => {
             res.status(404).json({ error: 'Order not found' });
         }
         await Order.updateOne({ _id: new ObjectId(orderId) }, { $set: { status: 'completed' } });
+        await sendEmail(order?.customerEmail, "Customer: Order Completed", "Your order ${orderId} has been completed.", `<p>Your order ${orderId} has been completed.</p>`)
+        await sendEmail(req.body.email, "Seller: Order Completed", "Order ${orderId} has been completed by ${order?.customerEmail}.", `<p>Order ${orderId} has been completed.</p>`)
```

Comment:
(Repeat) Seller email is not in `req.body.email`. Pull the user object from `itemOwnerId` of the order, then get their email from that.